.github/.wordlist.txt

```diff
@@ -618,6 +619,7 @@ idx
 ifconfig
 ifdef
 ifdefs
+ifx
```

Comment:
similar here and below. ifx and softwaretools are not words

Reply:
we have lots of words (even listed above that aren't words)